sd_hf_serve/app/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+docker logout
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-west-2.amazonaws.com
+docker pull 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference-neuronx:1.13.1-neuronx-py310-sdk2.15.0-ubuntu20.04
```

Comment:
The latest Neurno DLC is 2.18.0. Could you upgrade to the new one?

You can either use the ECR Public one (https://github.com/aws-neuron/deep-learning-containers?tab=readme-ov-file#pytorch-inference-neuronx) or private one (https://github.com/aws/deep-learning-containers/blob/master/available_images.md#neuron-containers)

Reply:
Updated to latest Neuron DLC is 2.18.0